runtime/jvmti/jvmtiFunctionTable.c

```diff
@@ -23,10 +23,24 @@
 #include "jvmtiHelpers.h"
 #include "jvmti_internal.h"
 
+#if JAVA_SPEC_VERSION >= 9
+#define JVMTI_MODULE_ENTRY(name) name
+#else /* JAVA_SPEC_VERSION >= 9 */
+#define JVMTI_MODULE_ENTRY(name) NULL
+#endif /* JAVA_SPEC_VERSION >= 9 */
+
+#if JAVA_SPEC_VERSION >= 11
+#define JVMTI_SAMPLING_ENTRY(name) name
+#else /* JAVA_SPEC_VERSION >= 11 */
+#define JVMTI_SAMPLING_ENTRY(name) NULL
```

Comment:
Can you rename `JVMTI_MODULE_ENTRY` to `JVMTI_9_ENTRY` and `JVMTI_SAMPLING_ENTRY` to `JVMTI_11_ENTRY`? That makes for a clearer pattern to follow for future versions.